### ISSUE TITLE
content covered by .compose-box-button-wrapper

### ISSUE THREAD

**Issue by noerw** [nolanlawson/pinafore#43]
Timeline content on the sameheight of as the floating compose button is not clickable. This is kinda confusing.
Thanks for this lightweight interface! :tada:

**Comment by nolanlawson** [comment]
Yup, I did this because of issues with sticky positioning in Chrome. I think I can come up with a workaround though.
